polygon-digital-carbon/src/RetirementHandler.ts

```diff
@@ -347,6 +347,16 @@ export function completeC3RetireRequest(event: EndAsyncToken): void {
         }
       }
 
+      // set status on retire as well
+      let retireId: Bytes | null = request.retire
+
+      if (retireId === null) {
```

Comment:
Are TypeScript object attributes `null` or `undefined` if they are not set?

I'm worried that `===` will not return `true` against an `undefined` value, but could just use `==` which will

![image](https://github.com/user-attachments/assets/eb773a96-4caa-4721-847b-b9ebe515d5b3)

![image](https://github.com/user-attachments/assets/9d6bb280-28b8-4486-ba75-e5844a8d4240)